config/300-apiserversource.yaml

```diff
@@ -38,12 +38,69 @@ spec:
   validation:
     openAPIV3Schema:
       properties:
-        apiVersion:
-          type: string
-        kind:
-          type: string
-        metadata:
+        registry:
           type: object
+          description: "Internal information, users should not set this property"
+          properties:
+            eventTypes:
+              # TODO: set the schemas.
+              type: object
+              description: "Event types that ApiServerSource can produce"
+              properties:
+                addResource:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                      pattern: "dev.knative.apiserver.resource.add"
+                    schema:
+                      type: string
+                      pattern: ""
```

Comment:
If we don't have a pattern, should we omit the field?

Reply:
I think it would be easier for the tooling to have both type and schema (and maybe some other CE attribute), i.e., each eventType within `registry.eventTypes` should have the same "core" structure. And that should be part of the Importer Spec. 

But you are right, let's remove the pattern from now, as it's empty.

Reply:
Done